inference-engine/src/vpu/myriad_plugin/myriad_config.cpp

```diff
@@ -44,6 +44,8 @@ IE_SUPPRESS_DEPRECATED_START
 
         VPU_MYRIAD_CONFIG_KEY(PLUGIN_LOG_FILE_PATH),
         VPU_MYRIAD_CONFIG_KEY(DEVICE_CONNECT_TIMEOUT),
+
+        VPU_MYRIAD_CONFIG_KEY(MOVIDIUS_DDR_TYPE),
```

Comment:
Could you please explain a bit what is the full option name?
Is it KEY_VPU_MYRIAD_MOVIDUS_DDR_TYPE? The `VPU_MYRIAD_MOVIDIUS` part looks excessive - is it reasonable to keep only `KEY_VPU_MYRIAD_DDR_TYPE` (or `VPU_MYRIAD_CONFIG_KEY(DDR_TYPE)` via macro)?

Reply:
Discussed offline